From 5ed28758a6d4ae23877e5699e9905380a9b56751 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Fri, 9 Aug 2024 23:59:28 +0530
Subject: [PATCH] Remove graphics mapper4

As IMapper4 is deprecated from Android15, mapper4 library removed.

Tests done:
- Android boot in GVT-d and BM config
- adb reboot

Tracked-On: OAM-122192
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 groups/device-specific/caas/product.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/groups/device-specific/caas/product.mk b/groups/device-specific/caas/product.mk
index 2d3ee1a..dc4b6ff 100755
--- a/groups/device-specific/caas/product.mk
+++ b/groups/device-specific/caas/product.mk
@@ -26,7 +26,6 @@ PRODUCT_PACKAGES +=  \
                     android.hardware.usb-service.example \
                     camera.vendor.device@1.0-impl \
                     android.hardware.camera.provider@2.4-impl-intel \
-                    android.hardware.graphics.mapper@4.0-impl.minigbm_intel \
                     android.hardware.graphics.allocator-service.minigbm \
                     android.hardware.renderscript@1.0-impl \
                     android.hardware.identity \
-- 
2.45.2

